tests/integration/common/data/test_freeze_time_fixture.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timedelta
+
+import pytest
+from sqlalchemy import select
+
+from app.common.data.models import MagicLink
+
+
+@pytest.mark.freeze_time("2025-01-01 13:30:00")
+def test_create_magic_link_frozen_time(db_session, factories, time_freezer):
+    factory_ml = factories.magic_link.create()
+    magic_link = db_session.scalar(select(MagicLink).where(MagicLink.id == factory_ml.id))
+    assert magic_link.created_at_utc == datetime.strptime("2025-01-01 13:30:00", "%Y-%m-%d %H:%M:%S")
```

Comment:
optional nit/suggestion (might just be my personal preference, do feel free to ignore): 

```suggestion
    assert magic_link.created_at_utc == datetime(2025, 1, 1, 13, 30, 0)
```